index.js

```diff
@@ -15,7 +16,7 @@ export default class ClapprStats extends ContainerPlugin {
   _inc(counter) {this._metrics.counters[counter] += 1}
   _timerHasStarted(timer){return this[`_start${timer}`] !== undefined}
   _start(timer) {this[`_start${timer}`] = this._now()}
-  _stop(timer) {this._metrics.timers[timer] += this._now() - this[`_start${timer}`]}
+  _stop(timer) {this[`_start${timer}`] ? this._metrics.timers[timer] += this._now() - this[`_start${timer}`] : 0}
```

Comment:
Yeah, this is very tricky to debug. And it only happens in `live` streams. I'll dig into this to try to find the real issue.